Camara Brasileira do Livro ISBN.js

```diff
@@ -0,0 +1,760 @@
+				"edition": "2",
+				"language": "pt-BR",
+				"libraryCatalog": "Câmara Brasileira do Livro ISBN",
+				"numPages": "232",
+				"place": "Rio de Janeiro",
+				"publisher": "FREITAS BASTOS EDITORA",
+				"shortTitle": "Manual de Sentença Trabalhista",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Direito"
+					},
+					{
+						"tag": "Elaboração"
+					},
+					{
+						"tag": "Jornada"
+					},
+					{
+						"tag": "Magistratura"
+					},
+					{
+						"tag": "Trabalhista"
+					},
+					{
+						"tag": "Trabalho"
+					},
+					{
+						"tag": "Técnica"
+					},
+					{
+						"tag": "concurso"
+					},
+					{
+						"tag": "juiz"
+					},
+					{
+						"tag": "modelos"
+					},
+					{
+						"tag": "sentença"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "search",
+		"input": {
+			"ISBN": "9786559602513"
+		},
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Batman",
+				"creators": [
+					{
+						"firstName": "John",
+						"lastName": "Ridley",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "James Tynion",
+						"lastName": "IV",
+						"creatorType": "editor"
+					},
+					{
+						"firstName": "Dandara",
+						"lastName": "Palankof",
+						"creatorType": "translator"
+					},
+					{
+						"firstName": "Pedro",
+						"lastName": "Catarino",
+						"creatorType": "illustrator"
+					},
+					{
+						"firstName": "Travel",
+						"lastName": "Foreman",
+						"creatorType": "illustrator"
+					},
+					{
+						"firstName": "Riccardo",
+						"lastName": "Federici",
+						"creatorType": "illustrator"
+					},
+					{
+						"firstName": "Jorge",
+						"lastName": "Jimenez",
+						"creatorType": "illustrator"
+					}
+				],
+				"date": "2022-03-11",
+				"ISBN": "9786559602513",
+				"abstractNote": "Aventuras do Batman",
+				"language": "pt-BR",
+				"libraryCatalog": "Câmara Brasileira do Livro ISBN",
+				"numPages": "100",
+				"place": "Barueri",
+				"publisher": "Panini Comics",
+				"attachments": [],
+				"tags": [
+					{
+						"tag": "Cartoons; caricaturas e quadrinhos"
+					},
+					{
+						"tag": "quadrinhos"
+					},
+					{
+						"tag": "super-herois"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "search",
+		"input": {
+			"ISBN": "9786559605101"
+		},
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Superman",
+				"creators": [
+					{
+						"firstName": "Sean",
+						"lastName": "Lewis",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Phillip Kennedy",
+						"lastName": "Johnson",
+						"creatorType": "editor"
```

Comment:
I don't know how much we can do here, but FWIW, Johnson is a co-author, here, not an editor
See e.g. https://www.martinsfontespaulista.com.br/superman---01-59---capa-variante-971290/p (but also just in general, he's a writer)